indico/web/client/styles/modules/_auth.scss

```diff
@@ -145,7 +145,7 @@
   }
 
   .titled-rule {
-    color: $gray;
+    color: #737373;
```

Comment:
how about `$light-black`? `#777` is very close to `#737373`

Reply:
Let's see with @tomasr8

Reply:
Close enough I guess, though firefox still doesn't like it (in fact it doesn't like any of the `$light-black` including the buttons and the footer). Maybe @foxbunny could also weigh in on this?

Reply:
what do you mean with "doesn't like it"? if `$gray` works i'd expect `$light-black` to work as well since both are defined in the same file... :o

Reply:
I meant in terms of accessibility i.e. the contrast with respect to the background

Reply:
ah ok, that makes more sense. 115 and 119 are so close so we could possibly just change the variable from `#777777` to `#737373` and nobody would notice the difference ;) (but please not in this PR)

Reply:
> (but please not in this PR)

Yes, please not in this PR. Ad-hoc changes just makes matters worse. Ideally we want to address all of these issues centrally in the design system.

Reply:
I'm going to remove the color change from this PR.